test/contract.js

```diff
@@ -21,38 +23,56 @@ describe('Contract', () => {
       assert.equal(contract.abi, ContractMetadata.EventFactory.abi);
     });
 
-    it('removes the hex prefix from the address', async () => {
+    it('removes the hex prefix from the address', () => {
       contract = new Contract(Config.QTUM_RPC_ADDRESS, '0x1234567890', ContractMetadata.EventFactory.abi);
       assert.equal(contract.address, '1234567890');
     });
   });
 
   describe('call()', () => {
-    it('returns the values', async () => {
+    it('returns the values', () => {
```

Comment:
This 'returns the values' name is blur to me. "Returns correct BN from <what result>"?